builder/azure/arm/config.go

```diff
@@ -392,11 +393,33 @@ func assertRequiredParametersSet(c *Config, errs *packer.MultiError) {
 	/////////////////////////////////////////////
 	// Capture
 	if c.CaptureContainerName == "" {
-		errs = packer.MultiErrorAppend(errs, fmt.Errorf("An capture_container_name must be specified"))
+		errs = packer.MultiErrorAppend(errs, fmt.Errorf("A capture_container_name must be specified"))
+	}
+
+	captureContainerNameRegex := regexp.MustCompile("^[a-z0-9][a-z0-9\\-]{2,62}$")
```

Comment:
Style tip: I put `regexp.MustCompile` declarations at package scope (not inside a function) because that way they will be checked at compile time instead of runtime. To follow the pattern used elsewhere this should be named `reCaptureContainerName`.